### PR TITLE
Add vsphere ovn-hybrid proxy workflow

### PR DESCRIPTION
**Pull request by sebsoto**
Allows easy creation of a proxy'd cluster for Windows through cluster-bot.

### CODE REVIEW

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml [hunk 1]
```diff
@@ -258,20 +258,7 @@ tests:
...
-    - ref: vsphere-deprovision-bastionhost
-    pre:
-    - chain: ipi-conf-vsphere
-    - chain: vsphere-provision-bastionhost
-    - ref: ipi-conf-proxy
-    - ref: ovn-conf
-    - ref: ovn-conf-hybrid-manifest-with-custom-vxlan-port
-    - chain: ipi-install-vsphere
-    test:
-    - ref: windows-e2e-operator-test-with-custom-vxlan-port
```
Comment (jrvaldes) [inline ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml:274]: Not following here, is the test being removed?

Comment (sebsoto) [reply ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml:274]: It should not be :facepalm: 

Thanks, I was wondering why pj-rehearse wasnt showing up

**Comment by sebsoto** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-vsphere-proxy-e2e-operator

**Pull request merged**